ldms/src/sampler/lustre_client/lustre_client_general.c

```diff
@@ -65,46 +65,64 @@ int llite_general_schema_is_initialized()
                 return -1;
 }
 
-int llite_general_schema_init(const char *producer_name)
+int llite_general_schema_init(comp_id_t cid)
 {
         ldms_schema_t sch;
         int rc;
         int i;
 
-        log_fn(LDMSD_LDEBUG, SAMP" llite_general_schema_init()\n");
+        log_fn(LDMSD_LDEBUG, SAMP ": llite_general_schema_init()\n");
         sch = ldms_schema_new("lustre_client");
-        if (sch == NULL)
+        if (sch == NULL) {
+		log_fn(LDMSD_LERROR, SAMP ": lustre_llite_general schema new failed (%s).\n",
```

Comment:
we have a bias against >80char lines.

Reply:
Yes, but there is consistency and there is foolish consistency. Sometimes a longer line is reasonable. But if you really want to avoid it, C concatenates string literals, so using that to split the message across lines would be preferable to using programmatic substitution.